cmake/Modules/platforms/Generic.cmake

```diff
@@ -8,7 +8,17 @@ macro (setGeneric)
     set(HOST_FLAG "" CACHE INTERNAL "Host Flag") ## default, no host_flag required
   endif()
   set(MKL_FLAG "-mkl"  CACHE INTERNAL "MKL Flag")
-  set(GSI_Intel_Platform_FLAGS "-DPOUND_FOR_STRINGIFY -O3 -fp-model source -assume byterecl -convert big_endian -g -traceback -D_REAL8_ ${OpenMP_Fortran_FLAGS} ${MPI_Fortran_COMPILE_FLAGS}" CACHE INTERNAL "GSI Fortran Flags")
+  set(GSI_Intel_Platform_FLAGS "${HOST_FLAG} -DPOUND_FOR_STRINGIFY -O3 -fp-model source -assume byterecl -convert big_endian -g -traceback -D_REAL8_ ${OpenMP_Fortran_FLAGS} ${MPI_Fortran_COMPILE_FLAGS}" CACHE INTERNAL "GSI Fortran Flags")
```

Comment:
Suggest to remove ${HOST_FLAG}. It causes problems in some computing platforms and does not improve optimization much.

Reply:
Hi, Actually, I copied this file from 'gsd/develop' branch. I removed this with new commit ("Removed $HOST_FLAG" 5f2ce50 )